pkg/platform/model/buildplanner.go

```diff
@@ -198,17 +195,28 @@ func (bp *BuildPlanner) FetchBuildResult(commitID strfmt.UUID, owner, project st
 	return &res, nil
 }
 
-func (bp *BuildPlanner) pollBuildPlan(owner, project, commitID string) (*bpModel.BuildPlan, error) {
+func (bp *BuildPlanner) pollBuildPlan(commitID string) (*bpModel.BuildPlan, error) {
 	var resp *bpModel.BuildPlan
 	ticker := time.NewTicker(pollInterval)
 	for {
 		select {
 		case <-ticker.C:
-			err := bp.client.Run(request.BuildPlan(owner, project, commitID), resp)
+			err := bp.client.Run(request.BuildPlan(commitID), resp)
 			if err != nil {
 				return nil, errs.Wrap(err, "failed to fetch build plan")
 			}
-			if resp.Project.Commit.Build.Status != bpModel.Planning {
+
+			if resp == nil {
+				continue
+			}
```

Comment:
This seems to be happening intermittently when we poll the build planner. I will follow up with PB to see if this is a bug.

Reply:
Still a concern?

Reply:
I've added a multilog call so we can know if this is happening. If we don't see it we can remove this. I've added extra error handling so hopefully this won't appear. In order to test properly I would have to disable it for a few integration test runs to see if it comes up.